pkgs/data/icons/iso-flags/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, perl
+, perlPackages
+, inkscape
+, pngcrush
+, librsvg
+, targets ? [ "aöö" ]
+}:
+
+stdenv.mkDerivation {
+  pname = "iso-flags";
+  version = "unstable-18012020";
+
+  src = fetchFromGitHub {
+    owner = "joielechong";
+    repo = "iso-country-flags-svg-collection";
+    rev = "9ebbd577b9a70fbfd9a1931be80c66e0d2f31a9d";
+    sha256 = "17bm7w4md56xywixfvp7vr3d6ihvxk3383i9i4rpmgm6qa9dyxdl";
+  };
+
+  nativeBuildInputs = [
+    perl
+    inkscape
+    librsvg
+  ] ++ (with perlPackages; [
+    JSON
+    XMLLibXML
+  ]);
+  
+  postPatch = ''
+    patchShebangs .
+  '';
+  
+  buildPhase = ''
+    make ${stdenv.lib.escapeShellArgs targets}
+  '';
```

I think you can just put `targets` in `makeFlags`  (or perhaps it's buildFlags, see setup.sh)

---

I'm overriding this in some places for cinnamon-screensaver so it's better to have this here, that way I'll save some headaches with the overrides (they're only using specific parts of that set - as the entire is too big)

---

I do not see how it would be any different with

```suggestion
  buildFlags = targets;
```